### ISSUE TITLE
login: ERROR missing paper key in login context

### ISSUE THREAD

**Issue by patdavid** [keybase/client#9497]
I installed the linux client on an Ubuntu 16.04 machine, but used a name I didn't want.

`keybase version 1.0.35-20171110170110+68245d46b`

I removed the device through the web interface and figured I'd log back in on the client again but use a new name.  Now when I try to log in:

```
Your keybase username or email address: USERNAME
patPlease enter the Keybase passphrase for USERNAME (6+ characters): 
In order to authorize this installation, keybase needs to sign this installation
with your GPG secret key SECRETKEY.

You have two options.

(1) Keybase can use GPG commands to sign the installation.

(2) Keybase can export your secret key from GPG and save it to keybase's local encrypted
    keyring. This way, it can be used in 'keybase pgp sign' and 'keybase pgp decrypt'
    going forward.
Which do you prefer?: 1


************************************************************
* Name your new device!                                    *
************************************************************



Enter a public name for this device: mother
▶ ERROR missing paper key in login context
```

Not sure what to do here?

**Comment by maxtaco** [comment]
Hi thanks for the report, this looks like a bug. Can you do a `keybase log send` and we'll check it out? Thank you. Cc: @patrickxb

**Comment by patdavid** [comment]
Because I’m an impatient idiot and technically know better, I reset my account and tried again with another new name, and it worked. :(

I can still generate the log if you want, but on this install it seems like it might me moot. Want me to create a new account to test this with and see if I can replicate?

**Comment by maxtaco** [comment]
It's OK, we'll see if we can figure it out. Definitely it's a bug on our part.

**Comment by jwitko** [comment]
Hi, I'm having this issue installing the windows client as well.  https://imgur.com/WwjEvzR
This happens as I try to login for the first time with the client after creating a new keybase account.  Here are clean log files used only for the failed login attempt:

keybase.service.log - https://gist.github.com/jwitko/c189881e3917c7149ea475e670c96ec5
keybase.app.log - https://gist.github.com/jwitko/07d494a4dd19ab4b4b60b5a63991f1f8
keybase.kbfs.log - https://gist.github.com/jwitko/a52a2eb77c28b0a567a96105260cd514

Currently this is blocking me from implementing a keybase based process within my company.

**Comment by maxtaco** [comment]
Cc @patrickxb

**Comment by maxtaco** [comment]
@jwitko i think the upshot is that you have to reset your account, since all of your active devices are revoked? I can’t check from my phone.

**Comment by patrickxb** [comment]
Yes, @jwitko you need to do an account reset for user `throtle`.

keybase.io/#account-reset

We have a fix in the next release that will prevent you from getting in the state that `throtle` is in, but right now, account reset will fix it.

Sorry for the inconvenience.

**Comment by jwitko** [comment]
@patrickxb @maxtaco i can still log into the account with my PC and encrypt/decrypt things.  Is it just non-browser devices that are revoked?

Resetting the account removes all keys as well right?  For example if I had already given the public key to someone to encrypt a file with would I still be able to decrypt after an account reset or would they have to r-encrypt with the new public key?  Mostly just curious on this question.

Thanks for the prompt responses, I will implement the work around.
